src/test/java/leave_room/LeaveRoomInteractorTest.java

```diff
@@ -31,27 +42,48 @@ public void setup() {
         leaveRoomInteractor = new LeaveRoomInteractor(roomDBGatewayMock, leaveRoomOutputBoundaryMock);
     }
 
+    /**
+     * Test case to verify that leaveRoom() successfully removes the user from the room's active user list
+     * and calls the leaveARoom() method on the LeaveRoomDBGateway mock.
```

Comment:
We don't have `leaveARoom` method in `LeaveRoomDBGateway` anymore. Checkout main branch.